manifests/init.pp

```diff
@@ -759,7 +759,7 @@
   $graphiteweb_log_dir_REAL         = pick($gr_graphiteweb_log_dir,    "${storage_dir_REAL}/log")
   $graphiteweb_conf_dir_REAL        = pick($gr_graphiteweb_conf_dir,   "${base_dir_REAL}/conf")
   $graphiteweb_webapp_dir_REAL      = pick($gr_graphiteweb_webapp_dir, "${base_dir_REAL}/webapp")
-  $graphiteweb_storage_dir_REAL     = $gr_graphiteweb_storage_dir
+  $graphiteweb_storage_dir_REAL     = pick($gr_graphiteweb_storage_dir, $gr_graphiteweb_storage_dir)
```

Comment:
sorry - mistake there. should have been $storage_dir_REAL... as config.pp was using $storage_dir_REAL as the directory for graphite.db. My thinking was that if you don't specify a custom location to the $graphiteweb_storage_dir then use $storage_dir_REAL.

Does that sound ok? if so I'll amend the commit and push back up.